frappe/public/js/frappe/form/form.js

```diff
@@ -1823,6 +1845,48 @@ frappe.ui.form.Form = class FrappeForm {
 			});
 		});
 	}
+
+	cancel_queued_action_interval() {
+		console.log('asdf')
```

Comment:
Should this (and similar instances) be `console.debug` or should be removed instead?